src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,31 @@
+.calendar {
+  display: flex;
+  padding: 10px;
+  justify-content: start;
+  flex-wrap: wrap;
+
+  &__day {
+    @include square($boxSize);
+    background-color: $backColor;
+    border: 1px solid black;
+    margin: 1px 1px 0 0;
+    transition: transform 500ms;
+
+    &:hover {
+      transform: translate(0, $shiftBox);
+      background-color: $backColorHov;
+      cursor: pointer;
```

Comment:
move `cursor: pointer;` to `&__day` selector

Reply:
Fixed